ui/src/containers/Details/index.tsx

```diff
@@ -4,13 +4,16 @@ import { useParams } from 'react-router-dom';
 import { Spinner } from '@patternfly/react-core';
 import { useMst } from '../../store/root';
 import BasicDetails from '../BasicDetails';
+import Description from '../../components/Description';
 
 const Details: React.FC = () => {
   const { resources } = useMst();
   const { name } = useParams();
 
   const updateVersions = () => {
     resources.versionInfo(name);
+    resources.readme(name);
```

Comment:
can you put in a separate function or rename the function `updateVersions` as general